src/geo-brand-presence/handler.js

```diff
@@ -44,7 +94,8 @@ export async function sendToMystique(context, getPresignedUrl = getSignedUrl) {
   // Get aiPlatform from the audit result
   const auditResult = audit?.getAuditResult();
   const aiPlatform = auditResult?.aiPlatform;
-  log.info('GEO BRAND PRESENCE: aiPlatform: %s', aiPlatform);
+  const providersToUse = aiPlatform !== undefined ? [aiPlatform] : WEB_SEARCH_PROVIDERS;
```

Comment:
You could do some additional verification here:
```suggestion
  const providersToUse = !isString(aiPlatform) ? [aiPlatform] : WEB_SEARCH_PROVIDERS;
```

```
import { isString } from '@adobe/spacecat-shared-utils';
```